src/lobster/assets/smiles_tokenizer/tokenizer_config.json

```diff
@@ -1,31 +1,31 @@
 {
   "added_tokens_decoder": {
     "0": {
-      "content": "<pad>",
+      "content": "<cls>",
```

Comment:
this will actually be very nice and convenient to have special tokens have shared IDs across tokenizers